erpnext/erpnext_integrations/doctype/backfill_taxjar_transactions/backfill_taxjar_transactions.json

```diff
@@ -0,0 +1,94 @@
+{
+ "autoname": "format: Transaction Import on {creation}",
+ "creation": "2021-11-10 03:20:52.588269",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "company",
+  "column_break_2",
+  "status",
+  "section_break_2",
+  "from_date",
```

Comment:
why have you created a different section for only two fields you can put them together.